src/RileyMaze/userFunction.js

```diff
@@ -240,6 +232,22 @@ ${code}
             return;
         }
 
+        if (scope.hasOwnProperty('riley') && scope.riley.queue.length !== MAX_QUEUE_LEN) {
```

Comment:
Likewise I'd prefer to avoid this special-case as well, if possible, because `if (scope.hasOwnProperty('riley'))` is basically saying `if (this is the "instructions" user function)`. Maybe it would work to add a `validate()` function to the descriptions in `USER_FUNCTIONS` similar to `getScope()`, and put this code there?